pyproject.toml

```diff
@@ -80,3 +80,4 @@ python_files = "test_*.py"
 
 [tool.poetry.scripts]
 sotopia_benchmark = "sotopia.benchmark:app"
+sotopia_install = "sotopia.install:app"
```

Comment:
it's more like a setup? rather than install.
Also if it is just redis, maybe call it redis?

Reply:
My goal is to make it simple to install sotopia. So ideally, we could install sotopia with `pip install sotopia; sotopia_install`. This PR focuses on Redis, and we can add new things in the future.

Reply:
hmmmm it still sounds like `sotopia_setup` to me

Reply:
I don't think setup is significantly different from install. 

1. Other widely used packages: E.g. this is similar to how `pre-commit` works: `pip install pre-commit; pre-commit install`. 
2. Wikipedia defines installation as synonym of setup: https://en.wikipedia.org/wiki/Installation_(computer_programs)

Reply:
Okay, that sounds good to me!